xcell/tests/data/dummy_signal_s0.txt

```diff
@@ -1 +0,0 @@
0611580e+03 -6.257718252128460335e+03 -1.231983685461106461e+03 -3.412930141075959000e+03 2.310653774850634363e+03 -3.733260442783946019e+03 -5.381969973449507734e+02 -1.488762172987799568e+04 -4.865019485425836137e+03 3.375692797111725667e+03 -5.489681030117365026e+02 1.419070020070055079e+03 1.730884398757864119e+03 -1.826268073165773842e+01 7.527546305149820682e+02 -1.066205226205022200e+03 1.224144326252343308e+04 3.278355892687563937e+03 -9.250114433675920509e+02 2.464883811681241923e+02 -3.015086943740008792e+03 6.636488906159214821e+03 -5.744363138173220250e+03 7.262465289688999292e+03 -1.163271406023565578e+04 1.763639663125119341e+03 3.904357874001531854e+03 3.706825500349178128e+03 -4.687884296571820414e+03 3.994073642036048568e+03 -4.968296764985329901e+03 -9.898917737538127994e+02 9.929587333839237544e+03 8.984130935936040032e+02 -1.631736463812676675e+03 -6.445416898086547917e+03 -2.817607173649672404e+03 2.481462056829022913e+03 8.179291454798283667e+03 2.905518107123313712e+03 4.419749802336291395e+03 -9.009970328943518325e+02 -3.183676981660740921e+02 -5.612766514584205879e+02 -1.234183481412794890e+03 8.442255361303499740e+03 -6.598460556494539560e+03 -3.302422714508626996e+03 -2.478730200404780135e+03 6.003954929127470450e+02 -9.785084273575884026e+02 3.166974712925682070e+03 -4.377109616918070060e+03 -1.071740069191314660e+03 -1.711555685081567617e+03 -5.178347562867682427e+03 -1.219857623101144554e+03 -4.108200758530284475e+03 -3.300512201638711304e+03 -5.677120754770973008e+02 4.038718781883242173e+03 7.276978236144233051e+03 6.854043907995652262e+03 -7.004748690280225674e+03 4.277071977559468905e+03 -8.441145428347959978e+02 -8.973601878261441016e+03 8.619268978038629712e+02 -2.105509003641195704e+03 4.159165344897492105e+03 -2.656220742473061364e+03 1.123162987510672110e+03 -6.888069731542754198e+03 3.985467270642748190e+03 -2.643705554149124737e+02 -4.227692233890466923e+03 -7.820497616991179939e+02 -5.844599856147821129e+02 -1.797215426932360060e+03 -7.037258947428048486e+03 1.401728123992197425e+03 -1.474143145265677231e+04 -6.371178492634079760e+01 1.048725373006413065e+03 5.935525258600833695e+03 -5.909448127296051098e+03 4.602084202252071009e+03 1.422256613163466454e+02 3.661508245163266110e+03 6.959643467596372375e+02 -1.532845141374571085e+03 4.497543151159768968e+03 7.247140031100448141e+03 2.921364251504031472e+03 1.187720969069174259e+01 7.606597655293765001e+02 -3.286968853711841803e+03 -2.482117145296751460e+03 9.049793694156196580e+03 -3.884947677395268784e+03 -1.084964489700927879e+04 -5.310324217228322595e+03 6.316575581967716062e+03 2.445186810807625079e+03 -1.101405104086147048e+03 -2.206279049047718672e+03 -1.363553092092693078e+03 -6.323092746166322513e+02 -5.707193471338815471e+03 6.962178678090395806e+03 -3.689385596800662825e+03 -2.053703256465581035e+03 -2.473288976855399142e+03 3.506147309107277579e+03 -1.221419591700916499e+03 1.558883943056544013e+03 -1.716447610605404179e+03 -2.594168397433187238
```

Comment:
Is this file empty. Shouldn't we have to remove it if so?

Reply:
It is not empty it the signal of the dummy mapper for a seed  = 0 . It has also been changed to a fits file that is more space efficient